blockchain/stake/internal/tickettreap/common.go

```diff
@@ -100,10 +117,63 @@ func nodeSize(node *treapNode) uint64 {
 
 // newTreapNode returns a new node from the given key, value, and priority.  The
 // node is not initially linked to any others.
-func newTreapNode(key Key, value *Value, priority int) *treapNode {
-	return &treapNode{key: key, value: value, priority: priority}
+func newTreapNode(key Key, value *Value, priority uint32) *treapNode {
+	return &treapNode{key: key, value: value, size: 1, priority: priority}
+}
+
+func (t *treapNode) getLeftSize() int {
```

Comment:
Please call this `leftSize`.  In general, `get` prefixes should be avoided per [Effective Go](https://golang.org/doc/effective_go.html#Getters) which the project follows per the [Code Contribution Guildelines](https://github.com/decred/dcrd/blob/master/docs/code_contribution_guidelines.md#ReqReading)

Also, all functions require comments.